script/1_0_2current_en_script/Arcueid/Day 2/02_00_ARC02_1B.txt

```diff
@@ -114,7 +114,7 @@ So, is this woman an acquaintance of my father?
 "Well, I've always been a carefree woman, like a webless spider drifting in the wind,
 but then fate brought me to Makihisa-kun.
 "If you have nowhere else to go, why don't you come live with us?"
-That's how cool he sounded when he said that! #But alas, He's not reaaaaaaally my type, so I simply turned him down, and so now I guess I'm friends with the current head? #Or something like that."
+That's how cool he sounded when he said that!# But alas, He's not reaaaaaaally my type, so I simply turned him down, and so now I guess I'm friends with the current head?# Or something like that."
```

Comment:
Not to self, verify this fixes the rendering issue

Reply:
Looks like it's a deepLuna problem not a game problem, will look into it
![2021-10-23-205421_360x79_scrot](https://user-images.githubusercontent.com/2476799/138575790-dc8d70c8-826c-4d7e-9fec-317b7acdc3bf.png)